source/infrastructure/instance-scheduler/lib/aws-instance-scheduler-stack.ts

```diff
@@ -51,6 +73,11 @@ export interface AwsInstanceSchedulerStackProps extends cdk.StackProps {
 */
 export class AwsInstanceSchedulerStack extends cdk.Stack {
 
+
+  readonly configurationTableOutput: CfnOutput;
+  readonly issueSnsTopicArn: CfnOutput;
+  readonly schedulerRoleArn: CfnOutput;
```

Comment:
for the pipeline to be able to access it (and thus publish it to the testing environment), it must be able to access the CfnOutput object.

This particular approach is a quick-and-dirty way of doing that, but just like with the cfn parameters comment I would be tempted to extract all of the CfnOutputs into some sort of consolidated object to improve this in the future.

Reply:
I'm saying that creating the `CfnOutput` explicitly may not be necessary. When you pass values across stacks, CDK creates outputs/parameters for you:

```ts
readonly configTableArn: string;
...
this.configTableArn = (coreScheduler.configTable.node.defaultChild as dynamodb.CfnTable).attrArn;
```

Again, not totally sure this will work, but it's worth trying.